src/lambda/commands/invokeLambda.ts

```diff
@@ -138,24 +129,19 @@ function createMessageReceivedFunc({
     // TODO: Consider passing lambdaClient: LambdaClient
     fn: LambdaFunctionNode // TODO: Replace w/ invokeParams: {functionArn: string} // or Lambda.Types.InvocationRequest
     outputChannel: vscode.OutputChannel
-    resourceFetcher: ResourceFetcher
-    resourcePath: string
     onPostMessage(message: any): Thenable<boolean>
 }) {
     const logger: Logger = getLogger()
 
     return async (message: CommandMessage) => {
         switch (message.command) {
             case 'sampleRequestSelected':
-                logger.info('selected the following sample:')
-                logger.info(String(message.value))
+                logger.info(`Requesting ${message.value}`)
+                const sampleUrl = `${sampleRequestPath}${message.value}`
 
-                const sample = await restParams.resourceFetcher.getResource([
-                    new WebResourceLocation(`${sampleRequestPath}${message.value}`),
-                    new FileResourceLocation(restParams.resourcePath)
-                ])
+                const sample = (await new HttpResourceFetcher(sampleUrl).get()) ?? ''
```

Comment:
I would have though the HTTP calls were handled by the `ResourceFetcher` itself - it seems we're inside the lambda invoker at this point... would this ultimately be moved out?

Reply:
The ResourceFetcher used earlier in this file gets the "sample data" manifest. Here we load the actual sample data payload that the user selected. I currently don't have any thoughts about how we'd change this, but I do know I want to add caching support in the future.